README.md

```diff
@@ -104,6 +104,19 @@ There is also a direct method, `compileStandard`, which is only present on recen
 
 Starting from version 0.4.20 a Semver compatible version number can be retrieved on every compiler release, including old ones, using the `semver()` method.
 
+#### From version 0.4.21
```

Comment:
Oh man, only if we'd have merged this at 0.4.21, it wouldn't be too big a breaking change anymore :)